app/domain/authentication/login.rb

```diff
@@ -14,41 +14,59 @@ module Authentication
   ) do
 
     def call
-      login(@authenticator_input)
+      validate_authenticator_exists
+
```

Comment:
You can remove all the blank spaces between these method calls. It's easy enough to read without the spaces.

Reply:
done